source/overview/compliance.rst

```diff
@@ -0,0 +1,89 @@
+
+Mattermost Enterprise Edition software is found under `ECCN 5D002 <https://www.bis.doc.gov/index.php/documents/regulations-docs/federal-register-notices/federal-register-2014/951-ccl5-pt2/file>`_, with a License Exception available of `“ENC” <https://www.bis.doc.gov/index.php/documents/regulation-docs/415-part-740-license-exceptions/file>`_, for our E10 and E20 software, with encryption features derived from open source software. Encryption products, under the export regulations, have multiple levels of controls and requirements. BIS has a separate section of their website that has an overview, and many links covering encryption, under `Policy Guidelines <https://www.bis.doc.gov/index.php/policy-guidance/encryption>`_ that you may want to review. These guidelines include helpful flow charts for determining if an item is subject to encryption controls, tables and other details.
+
+The other key areas to be aware of for export of Mattermost software or technology are:
+
+**Sanctions**: There are comprehensive sanctions to Cuba, Iran, North Korea, Sudan, Syria and other countries/territories
+with specific prohibitions, such as Russia and Venezuela, that you’ve probably been hearing about in the news. Details
+located here: `BIS <https://www.bis.doc.gov/index.php/forms-documents/regulations-docs/federal-register-notices/federal-register-2014/1063-746-1/file>`_ and `OFAC <https://www.treasury.gov/resource-center/sanctions/Pages/default.aspx>`_. The countries and sanctions are subject to change.
+
+**WMD (Weapons of Mass Destruction)**: Mattermost, its customers, and business partners may not export to parties involved
+in `proliferation <https://www.bis.doc.gov/index.php/documents/regulation-docs/413-part-736-general-prohibitions/file>`_ of weapons of mass destruction, along with other prohibited end-uses under the U.S. Export Administration Regulations (“EAR”).
+
+**General Prohibitions**: Information on General Prohibitions under the EAR are located `here <https://www.bis.doc.gov/index.php/forms-documents/doc_view/413-part-736-general-prohibitions>`_. Application of the applicability of these General Prohibitions is based on a combination of factors. Classification of the commodity, Destination, End-user, End-use and Conduct.
+
+**Restricted Parties**: You may not export to parties listed on US government `restricted parties lists <https://www.bis.doc.gov/index.php/policy-guidance/lists-of-parties-of-concern>`_, and should be screening against these prior to export. There is a `consolidated screening list <http://apps.export.gov/csl-search#/csl-search>`_ available supported by the U.S. government at export.gov at no charge, to use for screening.
+
+**Deemed Exports**:  Release of controlled technology to foreign persons in the U.S. are "deemed" to be an export to the
```

Comment:
Should this read "... Release of controlled technology........ is "deemed" to be an export..." ?